sale_stock_medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class MedicalPrescriptionOrderLine(models.Model):
+    _inherit = 'medical.prescription.order.line'
+
+    dispense_uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='Dispense UoM',
+        help='Dispense Unit of Measure',
+        required=True,
+        related='medicament_id.uom_id',
```

Comment:
You're right yeah my bad, no context involved. [Here](https://github.com/LasLabs/vertical-medical/blob/release/9.0/medical_prescription_sale_stock/models/medical_prescription_order_line.py#L67) though, I believe it's getting the current default uom of the product template model? That's what I meant to say.

If I don't set the dispense_uom_id to the related medicament's uom, I get ValidationError: Cannot convert uoms of different categories.